.gitlab/functional_test/static_quality_gate.yml

```diff
@@ -10,8 +10,6 @@ static_quality_gates:
   needs:
     - agent_deb-x64-a7
     - agent_deb-x64-a7-fips
-    - agent_deb-arm64-a7
```

Comment:
It feels like we should add the deb arm64 gates, instead of removing the dependency no ?

Reply:
The present fix is to remove unnecessary dependencies leading to extra wait time.

The more gates the Agent Build team adds, the more `needs` will increase. We will proceed gradually based on our tasks for the iteration. Please bear with us.

Reply:
I didn't want to add new static quality gates in this PR, but I'll sync with the team that owns them to see if they think that's something we should be doing

Reply:
Oh totally understood, we were discussing with Yoann which of the two options was the correct one anyway, thanks for the context !